### ISSUE TITLE
Support "Interlingua" locale

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-frontend#5878]
A new locale `ia` has been committed via Pontoon, it stands for "Interlingua": https://en.wikipedia.org/wiki/Interlingua.

For QA: "Interlingua" should be listed in the language chooser.

**Comment by vcarciu** [comment]
Verified as fixed : 
![interlingua](https://user-images.githubusercontent.com/1583842/43838828-1f9f05aa-9b25-11e8-80d5-deaca61b661c.gif)
